### PR TITLE
Update zhCN.lua

### PR DESCRIPTION
**Pull request by nanjuekaien1**
Fill in zhCN localization, add missing localization documents.

### CODE REVIEW

File: Dialog.lua [hunk 1]
```diff
@@ -22,7 +23,7 @@ StaticPopupDialogs['ASTRAL_KEYS_REFRESH_CONFIRM_DIALOG'] = {
 }
 
 StaticPopupDialogs['ASTRAL_KEYS_REFRESH_SUCCESS_DIALOG'] = {
-  text = "Refreshed key data.",
+  text = L['Refreshing key data.'],
```
Comment (Kyylier) [inline Dialog.lua:26]: Can we keep this the same? "Refreshed key data"

File: Lists/Friends.lua [hunk 2]
```diff
@@ -662,7 +663,7 @@ local function TooltipHook(self)
     local id = addon.UnitID(unit)
     if id then
     	GameTooltip:AddLine(' ')
-        GameTooltip:AddLine('Current Keystone')
+        GameTooltip:AddLine(L['CURRENT_KEY'])
```
Comment (Kyylier) [inline Lists/Friends.lua:666]: L['Current Keystone']

This goes in the tooltip and will look weird all capitalized.

Comment (nanjuekaien1) [reply Lists/Friends.lua]: correction

**Comment by Kyylier** [comment]
Thanks for the translations!

**Pull request merged**